### ISSUE TITLE
Спасибо за работу!

### ISSUE THREAD

**Issue by dimabendera** [dubovskiy/ria-ml#1]
Вы пошли в правильном направлении, с добавлением слоёв сверток. Конволюции могут имитировать некоторые функции нашего зрения, например мы можем смотреть на Кошку и сначала обращать внимание в целом на очертание кошки, потом на морду кошки, а потом на усы, нос - так же и модель, фактически первые слои это будут карты признаков общих очертаний, и самые последние это конкретные мелкие детали. Поэтому логично увеличивать после maxPool2d на каждой следующей конволюции количество наших карт признаков (filters: 32) - мелких деталей всегда больше.
Например:
<img src="https://www.researchgate.net/publication/328966158/figure/fig2/AS:693278764720129@1542301946576/An-overview-of-the-VGG-16-model-architecture-this-model-uses-simple-convolutional-blocks.png">
тут 64, потом 128 и так далее 
Так же правильно было добавить полносвязный слой dense после выхода из конволюции.

Ещё вы увеличили число эпох. Обычно пытаются сделать что бы модель сходилась как можно быстреее (за меньшее количество эпох) - это как минимум избавляет от ненужного ожидания. Если loss перестаёт уменшаться и прыгает - больше/меньше - ваша модель уже не обучается. Что бы избавится от впустую потраченного времени, обычно в модель добовляют колбек, который если ваш Loss не уменшается в течении N эпох обрывает обучение. Что бы модель быстрее сходилась - можно, например, просто подобрать более оптимальный learning rate.
Есть и обратная сторона, при большом количестве эпох может всё время падать loss, и модель будет показывать хорошие результаты на train датасете, но плохие на тесте. Это явление называется переобучением. Можно его представлять как "запоминание" данных, а не обучение на них. Есть соответственно разные механизмы для борьбы с переобучением, например можно добавить в модель слой Dropout.
Вот так переобучение и недообучение выглядит на графиках:
<img src="https://wiki.loginom.ru/images/overtraining.svg">

Спасибо за работу!
